src/mame/misc/marywu.cpp

```diff
@@ -14,6 +14,8 @@
     31 LEDs, 13 modules of double-digit 7-seg displays and 4 push-buttons).
   * we may also have user inputs from the coin slot and from the
     cabinet buttons, for making bets.
+// 09/07/2024
+psg fixed by flama12333.	
```

Comment:
We don't do this anymore, git file history is enough for such info. 
PR is otherwise ok, file needs some ironing out anyway ...

Reply:
If you know where it is then ok I guess?